### DISCUSSION TITLE
Menu高亮

### DISCUSSION THREAD

**Discussion by xuzhaohang** [tusen-ai/naive-ui#2582] [Q&A]
![1](https://user-images.githubusercontent.com/101177001/157206625-67940334-95ae-4524-9f3b-20aff3bc537b.png)
Edge和谷歌低版本浏览器，菜单Menu切换以后，切换前的菜单还是高亮

**Comment by 07akioni** [comment]
这个基本可以确认是浏览器的问题了，在新版可能会凑活处理一下

**Reply by 07akioni** [reply]
chrome 99 引入的
